linalg/slepc.hpp

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#ifndef MFEM_SLEPC
+#define MFEM_SLEPC
+
+#include "../config/config.hpp"
+
+#ifdef MFEM_USE_SLEPC
+#ifdef MFEM_USE_PETSC
```

Comment:
MFEM_USE_SLEPC should imply MFEM_USE_PETSC in the building process. No need to check it here